### PR TITLE
adding lgging framework to the app

### PR DESCRIPTION
**Pull request by bloodyIan**
Added electron-log to module, it's a clean framework, change of log path is still under work.

### CODE REVIEW

**Comment by bloodyIan** [comment]
In able to use the logging framework, you need to npm install electron-log first, log absolute path can be specified using log.file.file = <file path>

File: main.js [hunk 1]
```diff
@@ -20,6 +22,7 @@ function createWindow () {
 
   consoleWindow.on('closed', function () { consoleWindow = null, forWindow = null })
   forWindow.on('closed', function () { forWindow = null, consoleWindow = null })
+  log.info(':: Launching app');
```
Comment (somasundarambk) [inline main.js]: no need of ::

File: node_modules/electron-log/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Changelog
+
+**3.0.0-beta**
```
Comment (somasundarambk) [inline node_modules/electron-log/CHANGELOG.md]: Can you add the electron-log as a dependency?
npm install electron-log

File: main.js [hunk 3]
```diff
@@ -20,6 +22,7 @@ function createWindow () {
 
   consoleWindow.on('closed', function () { consoleWindow = null, forWindow = null })
   forWindow.on('closed', function () { forWindow = null, consoleWindow = null })
+  log.info(' Launching app');
```
Comment (somasundarambk) [inline main.js]: no space between ' and Launching

File: package.json [hunk 4]
```diff
@@ -13,6 +13,7 @@
     "jquery": "^3.4.1"
   },
   "devDependencies": {
-    "electron": "4.0.0"
+    "electron": "4.0.0",
+	  "electron-log": "3.0.7"
```
Comment (somasundarambk) [inline package.json]: align "electron" and "electron-log"
(same number of spaces before " and new line )

**Pull request merged**